src/ray/raylet/node_manager.cc

```diff
@@ -1035,6 +1039,8 @@ void NodeManager::ProcessDisconnectClientMessage(
                    << "job_id: " << worker->GetAssignedJobId();
   }
 
+  client->Close();
```

Comment:
Was this part of the bug?

Reply:
Yes, this was another bug where we were running out of file descriptors.